### PR TITLE
Added loft-list for json to html table display

### PR DESCRIPTION
**Pull request by vvaldez**
* Also modified loft-launch to support new tags
* Now tagging instances with lab_type and lab_user
* Also added output for instance creation count

/cc @scollier @tchughesiv

### CODE REVIEW

File: aws-cli/loft-list [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#!/bin/bash -x
+
+LIST_FILE=aws-loft-list.json
+TAG_KEY=lab_type
+TAG_VALUE=loft-lab
+
+aws ec2 describe-instances --query 'Reservations[].Instances[].{PublicHostname:PublicDnsName,PublicIP:PublicIpAddress}' --filters "Name=instance-state-name,Values=running" "Name=tag:${TAG_KEY},Values=${TAG_VALUE}" --output json | jq --arg START 1 '($START | tonumber) as $s
+  | to_entries
+  | map({StudentID: ($s + .key), PublicHostname:.value.PublicHostname, PublicIP:.value.PublicIP })' > ${LIST_FILE}
```
Comment (scollier) [inline aws-cli/loft-list]: I tried out "Student ID" and "Public Hostname", and "Public IP" (note the spaces), and it worked for the text based json output. Not sure though, how that would translate into html though.

Comment (vvaldez) [inline aws-cli/loft-list]: @scollier did you want me to test with spaces? Is that for readability?

**Comment by vvaldez** [comment]
Rebased, merging.

**Comment by vvaldez** [comment]
@scollier I don't have write access

**Pull request merged**